src/packages/@ncigdc/containers/explore/ClinicalAggregations.tsx

```diff
@@ -67,7 +71,7 @@ const facetMatchesQuery = (
 ): boolean => {
   return _.some(
     [
-      _.replace(facet.field.split('.')[1], /_/g, ' '),
+      _.replace(facet.field.split('.').pop() || '', /_/g, ' '),
```

Comment:
the problem comes from pop(). For [].pop(), it will returen `undefined` instead of '' So I have to pu a '' there.

Reply:
If you call split with a character that isn't in the string, it just returns as `["string"]`. I think if you want to account for `facet.field` not having a value it would be `(facet.field || '').split('.')`